src/redux/Books/Books.js

```diff
@@ -0,0 +1,23 @@
+const ADD_BOOK = 'BookStore/Books/ADD_BOOK';
+const REMOVE_BOOK = 'BookStore/Books/REMOVE_BOOK';
+
+const BookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {
+        ...state,
+        books: [state.title, state.author, action.type],
+      };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        books: state.filter((book) => book.title !== action.type),
```

Comment:
- [ ] Nice work removing the book using the filter method but Kindly the condition of filtering is wrong, the best way is to target the id because every book has a unique id so you can do it like that:
`state.filter(book => book.id !== id);`